### ISSUE TITLE
Plugin elastalert-kibana-plugin [7.5.0] is incompatible with Kibana [7.9.3]

### ISSUE THREAD

**Issue by psganeshk** [bitsensor/elastalert-kibana-plugin#160]
Plugin elastalert-kibana-plugin [7.5.0] is incompatible with Kibana [7.9.3]

**Comment by nsano-rururu** [comment]
7.5.1～7.8.1
https://github.com/nsano-rururu/elastalert-kibana-plugin/releases/tag/1.2.0
7.9.0 ～ 7.9.3
https://github.com/nsano-rururu/elastalert-kibana-plugin/releases/tag/1.3.0

**Comment by nsano-rururu** [comment]
By the way, the current elastalert-kibana-plugin cannot work with the latest kibana 7.10.0.

**Comment by nsano-rururu** [comment]
Please close

**Comment by ankitdevnalkar** [comment]
Do you have any plan to release compatible version with Kibana 7.10.0  ?

**Comment by nsano-rururu** [comment]
I can't promise how long. I'll do it if I have time.

**Comment by nsano-rururu** [comment]
@ankitdevnalkar

The reason I have time is because I've been the maintainer of praeco and elastalert-server since August of this year, so I've been spending time on issues, bug fixes, and feature additions for praeco and elastalert-server.
https://github.com/johnsusek/praeco
https://github.com/johnsusek/elastalert-server

**Comment by ankitdevnalkar** [comment]
No worries @nsano-rururu . Just wanted to check if it will be available in near future or not. :slightly_smiling_face:

**Comment by ankitdevnalkar** [comment]
hey buddy @nsano-rururu ! just curious if it will be available within 1-2 months, else I can start searching for alternatives.

**Comment by nsano-rururu** [comment]
@ankitdevnalkar

> hey buddy @nsano-rururu ! just curious if it will be available within 1-2 months, else I can start searching for alternatives.

If so, I think you should consider other things.

**Watcher**

https://www.elastic.co/guide/en/kibana/current/watcher-ui.html

**Alerting and action settings in Kibana**

[Alerts and Actions](https://www.elastic.co/guide/en/kibana/current/managing-alerts-and-actions.html)
[Managing Alerts](https://www.elastic.co/guide/en/kibana/current/alert-management.html)
[Alert details](https://www.elastic.co/guide/en/kibana/current/alert-details.html)
[Managing Connectors](https://www.elastic.co/guide/en/kibana/current/connector-management.html)

**Open Distro for Elasticsearch**

[Alerting](https://opendistro.github.io/for-elasticsearch-docs/docs/alerting/)

**Grafana**

Perform alerting on data in Elasticsearch with Grafana
https://grafana.com/docs/grafana/latest/alerting/
https://grafana.com/docs/grafana/latest/alerting/notifications/

**Comment by ankitdevnalkar** [comment]
Yeah I am checking Open Distro. I am asking because I have 1-2 months to push my project in production, I thought it would be great to have ElastAlert along side.

**Comment by nsano-rururu** [comment]
@ankitdevnalkar 
Currently, it is unclear whether the maintainers of elastalert will be replaced, and we do not recommend adopting elastalert because it has not been actively added or modified for a long time. If something goes wrong, you need to be prepared to fix the bug yourself.

**Comment by nsano-rururu** [comment]
@ankitdevnalkar 
And because I'm not the official maintainer of the kibana plugin. I don't have time at all right now and the latest kibana is not supported. It's clear that the program is likely to be completely refurbished

**Comment by nsano-rururu** [comment]
@ankitdevnalkar

I received a pull request for kibana 7.10, so I will merge it and release it if there is no problem after checking the operation.
[Update to Kibana 7.10 #125](https://github.com/nsano-rururu/elastalert-kibana-plugin/pull/125)

**Comment by ankitdevnalkar** [comment]
Thanks for the much-awaited update @nsano-rururu !!

**Comment by nsano-rururu** [comment]
@ankitdevnalkar

I'm asking a question because the build doesn't pass. It will take some time to release.

**Comment by nsano-rururu** [comment]
@ankitdevnalkar

It seems that it will take some time to release because the setting of kibana.yml has not been read yet, only it can be displayed in kibana. Work on the following branches:
https://github.com/nsano-rururu/elastalert-kibana-plugin/tree/migrating-plugin-new-platform

**Comment by nsano-rururu** [comment]
@ankitdevnalkar

I haven't confirmed that it works properly because it is not supported, but it seems that there is a release that supports kibana 7.10. Why don't you give it a try?
https://github.com/ch-bas/elastalert-kibana-plugin

**Comment by nsano-rururu** [comment]
@ankitdevnalkar

The following repositories that I contacted the other day that support kibana 7.10 do not seem to work properly, so it seems better not to use them.
https://github.com/ch-bas/elastalert-kibana-plugin

**Comment by nsano-rururu** [comment]
@ankitdevnalkar

I contacted the person who responded and asked me to fix it. Is the setting information an environment variable from kibana.yml? Please note that it seems to have changed to.
https://github.com/ch-bas/elastalert-kibana-plugin

**Comment by haodketernal** [comment]
Have you release this plugins ?

**Comment by nsano-rururu** [comment]
@haodketernal

look
https://github.com/bitsensor/elastalert-kibana-plugin/issues/166

**Comment by nsano-rururu** [comment]
bitsensorp/elastalert-plugin is no longer maintained.

**Comment by nsano-rururu** [comment]
Ask a question in the repository you are maintaining
https://github.com/Karql/elastalert-kibana-plugin
